### PR TITLE
Added [Wordpress] theme support

### PR DESCRIPTION
**Pull request by bgun**
Added badges for Wordpress theme downloads (total) and theme rating.

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -4844,6 +4845,70 @@ cache(function(data, match, sendBadge, request) {
   });
 }));
 
+// wordpress theme rating integration.
+// example: https://img.shields.io/wordpress/theme/r/hestia.svg for https://wordpress.org/themes/hestia
+camp.route(/^\/wordpress\/theme\/r\/(.*)\.(svg|png|gif|jpg|json)$/,
+cache(function(data, match, sendBadge, request) {
+  var theme = match[1];  // eg, `hestia`.
+  var format = match[2];
+  var apiUrl = 'https://api.wordpress.org/themes/info/1.1/?action=theme_information&request[slug]=' + theme;
```
Comment (paulmelnikow) [inline server.js]: Are theme names always alphanumeric? It's probably safer to use the URL-encode `theme`, or better yet, use the `query` option to `request`. Some of the other services do this.

**Comment by paulmelnikow** [comment]
Fixes #869.

**Comment by bgun** [comment]
No problem! Added validation using the 'querystring' library in accordance with other services. Also added tests for all 6 Wordpress integrations.

**Comment by bgun** [comment]
Also, test would not run without installing `joi` and `minimist` - not sure if that was intentional, but added them to `dev-dependencies`.

**Comment by paulmelnikow** [comment]
Thanks!

`minimist` should definitely be added to the dev dependencies. It's used directly by our test runner.

`joi` gets installed by icedfrisby and we should make sure only one version of it gets installed. If possible I'd rather avoid directly adding it until MarkHerhold/IcedFrisby#52 is resolved.

Which node / npm version are you using?

**Comment by bgun** [comment]
Removed joi. I'm using Node 6.9.2, npm 3.10.9

**Comment by paulmelnikow** [comment]
So strange! I was on Node 6 / npm 3 and didn't have the problem then, nor after I upgrade.

Thanks. I'll try to tackle MarkHerhold/IcedFrisby#52 so this can be resolved more satisfactorily.

**Pull request merged**